spec/models/partnership_spec.rb

```diff
@@ -75,7 +75,8 @@
     end
 
     it "returns false when the challenge_deadline is blank" do
-      partnership = create(:partnership, challenge_deadline: nil)
+      partnership = create(:partnership)
+      partnership.update!(challenge_deadline: nil)
```

Comment:
Why does this need to change?
```suggestion
      partnership = create(:partnership, challenge_deadline: nil)
```